molgenis-api-metadata/src/test/resources/l10n/api-metadata_en.properties

```diff
@@ -1 +1,4 @@
-MAPI01=Query ''{0}'' didn't lead to any results
\ No newline at end of file
+MAPI01=Query ''{0}'' didn''t lead to any results.
+MAPI02=An entityType cannot have zero attributes.
+MAPI03=It is not allowed to edit the identifier of an attribute.
+MAPI04=Field ''{0}'' is unknown for ''{1}'' requests.
```

Comment:
I was trying to update the `backend` and `tags` fields and I think this message is a bit confusing: `"Field 'tags' is unknown for 'entityType' requests."`

In these cases I'd expect something like `"Updating field 'backend' of 'entityType' is not allowed"`
In the cases where the field really doesn't exist maybe an `UnknownAttributeException` would suffice.

_BTW:_ This exception also isn't localised on the preview server, haven't tried locally yet.

Reply:
Let's discuss this, since we designed the api not to know those fields its hard to make an exception in the exception that gets thrown here...

Reply:
Nevermind, we can't get away without specific code for these fields in other places as well, I'll create an specific exception for this